web/impact/impact/tests/factories/accelerator/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .startup_factory import StartupFactory
+from .currency_factory import CurrencyFactory
+from .industry_factory import IndustryFactory
+from .organization_factory import OrganizationFactory
+from .recommendationtag_factory import RecommendationTagFactory
+from .jobposting_factory import JobPostingFactory
```

Comment:
thats a good question - my view was that the accelerate system considers them to be totally separate objects (and on initialization of the actual models I'm sure there are some minute internal differences like references to app_name etc) and so should have their own factories. here. open to potentially reusing the impact factories for the accelerator app as well.

Reply:
I don't entirely understand you answer here. But generally speaking, if there's a dilemma on an approach to take, then feel free to raise it up on slack or after a standup. If you're asking my opinion, then I don't currently see a compelling reason to not put our factories in  django-accelerator, since I don't see how cases where we'd _want_ them to be diverge across code-bases. Any discrepancy that we have now I'm seeing as a temporary step, which we should not extend. Obviously, if you see such reasons to keep them apart, I'd love to hear them.